overviewer_core/world.py

```diff
@@ -200,8 +200,8 @@ def find_true_spawn(self):
         ## clamp spawnY to a sane value, in-chunk value
         if spawnY < 0:
             spawnY = 0
-        if spawnY > 255:
-            spawnY = 255
+        if spawnY > 319:
+            spawnY = 319
```

Comment:
Maybe the `spawnY < 0` check should also be updated to `-64`?

Reply:
Excellent point. I forgot to take that into account. Let me fix that.